### DISCUSSION TITLE
Where are the logstash logs or conf.d located?

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#10161] [Unsupported Versions] [answered]
Hi,
  I am using:
  securityonion-2.3.190
  windows 10 

 I have looked through several google searches. I keep getting locations where I look and its not there. I am looking for logstash.yml, conf.d and any other logstash config's. I am trying to see why my sysmon data dosnt get to kibana in the SOC.

/etc/logstash/conf.d  (logsearch not found)

/opt/so/saltstack/local/pillar/ 

/opt/so/saltstack/local/salt/logstash (no logstash config file)

 I see folders but nothing in them, especially conf.d

thanks for any help or advice

**Comment by cm-ops** [comment]
Take a look here https://docs.securityonion.net/en/latest/directory.html? But those specific logs are at `/opt/so/log/logstash/logstash.log` and `/opt/so/conf/logstash/etc/logstash.yml`.

**Reply by iqworks** [reply]
Hi cm-ops, i was on another project for the last couple of weeks. . 
thanks for the reply. But I cannot see logstash? 
![image](https://user-images.githubusercontent.com/5569263/236640291-7472a5bb-406a-4c63-82c1-5828859389ad.png)

I dont see it here either?  /etc/logstash/conf. d
https://groups.google.com/g/security-onion/c/4FwPbMqmuEM 
![image](https://user-images.githubusercontent.com/5569263/236640449-aee0386b-c421-497c-9160-db5d57e588e8.png)

thanks for any suggestions or advice

**Comment by dougburks** [comment]
@iqworks Per your screenshot above, if the `/opt/so/conf/` directory does not contain a `logstash` sub-directory, then that most likely means that you are running Security Onion in either `Eval` or `Import` mode. From https://docs.securityonion.net/en/2.3/logstash.html:

_When Security Onion 2 is running in Standalone mode or in a full distributed deployment, Logstash transports unparsed logs to [Elasticsearch](https://docs.securityonion.net/en/2.3/elasticsearch.html#elasticsearch) which then parses and stores those logs. It’s important to note that Logstash does NOT run when Security Onion is configured for Import or Eval mode. You can read more about that in the [Architecture](https://docs.securityonion.net/en/2.3/architecture.html#architecture) section._

**Reply by iqworks** [reply]
Thanks Doug. how can I tell which mode I selected during setup? I wanted to do that before re-running sosetup. I looked here 
https://docs.securityonion.net/en/2.3/help.html - it said "Setup /root/sosetup.log"? Is this where I can see how I set up this VM?
I went here but didnt find it? 
![image](https://user-images.githubusercontent.com/5569263/236862780-6aa7d29c-c001-4930-b411-54e715a293e9.png)

Am I looking in the right place for this sosetup.log?
 
thanks again Doug

**Reply by iqworks** [reply, marked as answer]
my bad, I went here and see it now
![image](https://user-images.githubusercontent.com/5569263/236887838-370b9514-e65a-40e2-b781-7ab92f95d9a8.png)

I see this
![image](https://user-images.githubusercontent.com/5569263/236889099-adb60ab7-b9af-4673-8c2b-11e4f6862b3d.png)

I am going to run rerun sosetup.
